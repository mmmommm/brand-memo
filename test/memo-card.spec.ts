import MemoCard from "@/components/molecules/memo-card.vue"
import Vuetify from "vuetify"

import { createLocalVue, shallowMount } from "@vue/test-utils"

describe("memo-card.vue", () => {
  const localVue = createLocalVue()
  let vuetify

  beforeEach(() => {
    vuetify = new Vuetify()
  })
  
  const wrapper = shallowMount(MemoCard, {
    localVue,
    vuetify,
    propsData: {
      memo: {
        code: 1111,
        slug: "11111111",
        name: "dummy",
      }
    }
  })
  it("<MemoCard />", () => {
    expect(wrapper.element).toMatchSnapshot()
  })
})
