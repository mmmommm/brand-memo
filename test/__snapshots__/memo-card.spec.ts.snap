// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`memo-card.vue <MemoCard /> 1`] = `
<v-flex-stub
  md4=""
  sm6=""
  tag="div"
  xs12=""
>
  <v-btn-stub
    activeclass=""
    class="my-5"
    height="140"
    outlined="true"
    tag="button"
    to="[object Object]"
    type="button"
    width="350"
  >
    <v-card-stub
      height="140"
      loaderheight="4"
      tag="div"
      width="350"
    >
      <v-card-text-stub
        class="ml-2 code"
      >
        
        
      
      </v-card-text-stub>
       
      <v-card-title-stub
        class="ml-2 name"
      >
        
        
      
      </v-card-title-stub>
    </v-card-stub>
  </v-btn-stub>
</v-flex-stub>
`;
